apps/sports-helsinki/src/pages/search/index.tsx

```diff
@@ -74,6 +77,11 @@ export async function getStaticProps(context: GetStaticPropsContext) {
         id: `/${language}${ROUTES.SEARCH}/`,
       },
       fetchPolicy: 'no-cache', // FIXME: network-only should work better, but for some reason it only updates once.
+      context: {
+        headers: {
+          authorization: isPreview ? `Bearer ${previewData?.token}` : '',
+        },
+      },
```

Comment:
If there is no token at all, we should not have the authorization header either.

Reply:
I guess that changing the default from `""` to `undefined ` could do the trick, but it would probably be better if the whole header context would be set conditionally.

Reply:
Now when you hvae added an utility lib for jwtToken, would it be better to prefix the token with the "Bearer" there in that util?

Reply:
the new implementation is much better